tests/modules/site-health/audit-full-page-cache/audit-full-page-cache-test.php

```diff
@@ -0,0 +1,367 @@
+<?php
+/**
+ * Tests for audit-full-page-cache module.
+ *
+ * @package performance-lab
```

Comment:
Can you add a group here with the module slug?
```suggestion
 * @package performance-lab
 * @group audit-full-page-cache
```